### ISSUE TITLE
Wizard vs Dock - Volcano Point IF

### ISSUE THREAD

**Issue by Charlotte-Morgan** [AIFDR/inasafe#2091]
# problem
I can set the key words on volcano point and buildings and use the wizard to run an impact function however, there are no valid IF according to the dock
![image](https://cloud.githubusercontent.com/assets/5743135/8538393/ded3232a-2495-11e5-8ee8-44e9ee262983.png)

**Comment by ismailsunni** [comment]
The reason of this issue, when we assigned keyword to volcano IF, we doesn't specify the classification. Meanwhile, in the volcano IF, there is layer requirement for hazard layer, `'vector_hazard_classification': 'volcano_vector_hazard_classes'`

Alternative solutions:
- Assign the missing keyword under the hood
- Remove the 'volcano_vector_hazard_classes' requirement from the IF.


I prefer the second solution, since we do not give magic in the wizard. Moreover, it's more logical since for volcano IF, we do not need a field that represent the classes.

I made PR #2097 for solving this ticket by using the second solution.
